src/test/librados/io_cxx.cc

```diff
@@ -218,6 +218,38 @@ TEST_F(LibRadosIoPP, SparseReadOpPP) {
   }
 }
 
+TEST_F(LibRadosIoPP, SparseReadExtentArrayOpPP) {
+  char buf[128], zbuf[128];
+  memset(buf, 0xcc, sizeof(buf));
+  memset(zbuf, 0, sizeof(zbuf));
+  bufferlist bl;
+  int i, len = 1024, skip = 5;
+  bl.append(buf, sizeof(buf));
+  for (i = 0; i < len; i++) {
+    if (!(i % skip)) {
+      ASSERT_EQ(0, ioctx.write("sparse-read", bl, bl.length(), i * sizeof(buf)));
+    }
```

Comment:
This can be expressed more directly:
```suggestion
  for (i = 0; i < len; i += skip) {
    ASSERT_EQ(0, ioctx.write("sparse-read", bl, bl.length(), i * sizeof(buf)));
```

Reply:
Done. Fixed it.